specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -271,6 +402,171 @@
         }
       }
     },
+    "MultiLanguageConversationAnalysisInput": {
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Conversation"
+          }
+        }
+      }
+    },
+    "Conversation": {
+      "required": [
+        "id",
+        "language",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "description": "Identifier for the conversation",
+          "type": "string"
+        },
+        "language": {
+          "description": "Default languages of the conversation item in BCP-47 format",
+          "type": "string"
+        },
+        "modality": {
+          "$ref": "#/definitions/Modality"
+        },
+        "domain": {
+          "type": "string"
+        },
+        "conversationItems": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationItem"
+          }
+        }
+      }
+    },
+    "Modality": {
+      "type": "string",
+      "description": "Enumeration of supported conversation genres (or modalities). Currently we will support chat and transcription",
+      "enum": [
+        "transcript",
+        "text"
+      ],
+      "x-ms-enum": {
+        "name": "Modality",
+        "modelAsString": true
+      }
+    },
+    "ConversationItem": {
+      "discriminator": "modality",
+      "required": [
+        "id",
+        "participantId",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique identifier for the conversation item."
+        },
+        "participantId": {
+          "type": "string",
+          "description": "Identifier for the participant."
+        },
+        "language": {
+          "type": "string",
+          "description": "Optional property to override the language of the conversation item in BCP-47 format."
+        },
+        "modality": {
+          "description": "Property to override the modality of the conversation item.",
+          "$ref": "#/definitions/Modality"
+        }
+      }
+    },
+    "TextConversationItem": {
```

Comment:
Just a thought: The more I think about it the more I think we should just rephrase "Text" and "Transcripts" to "Written" and "Spoken".

Reply:
Yes I initially used "Spoken". However, I am not sure who but we had a confusion on whether "Spoken" can be construed as audio so instead we used "transcript". Matthew had also suggested "transcribedText"?

Reply:
there will be always nuances, i am not sure spoken and text will get that resolve either.

Reply:
For now we are using "text" and "transcript" as the 2 modalities. We can update this in next preview based on customer feedback.